app/views/content/train-to-be-a-teacher/_why-teach.html.erb

```diff
@@ -0,0 +1,26 @@
+<section class="why-teach" aria-label="Why teach?">
+  <div class="why-teach__container">
+    <div class="why-teach__content">
+      <h2>
+        <div class="caption">Why teach?</div>
+        Inspire the next generation
+      </h2>
+
+      <p>
+        The impact of your work as a teacher will be felt beyond the classroom
+        walls and far into the future. Teachers make a difference every day.
+      </p>
+
+      <ul>
+        <li><%= link_to("A day in the life", "/a-day-in-the-life-of-a-teacher") %></li>
+        <li><%= link_to("Benefits of teaching", "/salaries-and-benefits") %></li>
```

Comment:
@valentine-carter Is this link text a bit misleading until we have more Why teach content? It just goes to the salaries page which doesn't seem to quite fit with the Inspire the next generation message of this section?

Reply:
Indeed. I need to change the copy on this, it's still the mock copy. I just need to fix my ports not loading first.